style-guide/unorder-list.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
```

Comment:
We should think about if there is some way to intelligently guess whether a list looks like procedural steps or if it looks like a collection of things. I'm not sure... 
Off the top of my head, I would think looking for verbs and verb cases. But that might be too hard to actually do.

Reply:
I searched for this but I was not able to find a suitable way to do this. Maybe we should remove the checks for lists because as of now we have all lists in the proper format and for new PRs we can manually check the lists. It would be better to this rather than generating a list of warnings for all lists.